source/JsMaterialX/README.md

```diff
@@ -148,42 +149,6 @@ Documents can be written to strings via the `writeToXmlString` method, and to fi
 
 Note that the `XmlWriteOptions.elementPredicate` option is not supported in JavaScript.
 
-## Maintaining the Bindings
```

Comment:
Don't get confused by this. There are no changes below this point, except for moving the shader gen section to a different place.